graphql/custom_scalars.go

```diff
@@ -6,9 +6,12 @@ import (
 	"io"
 
 	"github.com/99designs/gqlgen/graphql"
+	restModel "github.com/evergreen-ci/evergreen/rest/model"
 	"github.com/mongodb/grip"
 )
 
+type JIRANotificationsProjectMap map[string]restModel.APIJIRANotificationsProject
+
```

Comment:
This seems to specific to be a `scalar`. Maybe we can consider using a different type and handle that in the generated resolver